integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html

```diff
@@ -55,12 +58,7 @@
         />
       </form>
     </ng-container>
-    <ng-container
-      *ngIf="
-        paymentLink.data &&
-        paymentLink.renderType === RENDER_PATTERN.HOSTED_FIELDS
-      "
-    >
+    <ng-container *ngIf="paymentLink.data && !paymentLink?.destination?.url">
```

Comment:
all patterns inject HTML except the ones that have destination.url.
On workbench Iframe and FullPage both have the 2 options 'Create by script' & 'Create by Url'